<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyBATDialog</class>
 <widget class="QDialog" name="CopyBATDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakeBAT</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>BATファイル名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBATFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CopyAll.bat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectBATFile">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GUI path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGUI">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Algorithm path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAlgorithm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPIOFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PIO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCameraFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cameral</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSequenceFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SequenceDLL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGeneralStockerFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GeneralStocker</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFilter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGlobalDat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Global.datファイル</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectGlobalDat">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditResultFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ResultDLL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLightFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LightDLL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMakeBAT">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>760</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Bat生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>760</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelecGUIFolder">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectAlgorithmFolder">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectPIOFile">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectCameraFile">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectLightFile">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectSequenceDLL">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectGeneralStockers">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectFilterPath">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectResultDLL">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveSetting">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>760</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadSetting">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>790</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load setting</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetAdditional">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <column>
    <property name="text">
     <string>Source File</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destination</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>追加ファイル</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlus">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>610</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMinus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>650</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxInspectionPlayer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>InspectionPlayer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxInspectionEditor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>InspectionEditor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRepairGUIMain">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RepairGUIMain</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
